maven-model/src/test/java/org/apache/maven/model/BuildTest.java

```diff
@@ -53,4 +53,12 @@ public void testToStringNullSafe()
         assertNotNull( new Build().toString() );
     }
 
+    public void testToStringNotNonsense()
+    {
+        Build build = new Build();
+
+        String s = build.toString();
+
+        assert "Build(super=BuildBase(super=PluginConfiguration(super=PluginContainer())))".equals( s ) : s;
```

Comment:
That's a very verbose and not really useful `toString()` ...

Reply:
> The toString() result should at least have the simple class name in it.

Please suggest what else could be included, or what to leave.

Reply:
> That's a very verbose and not really useful `toString()` ...

Also - I've stolen the format from https://projectlombok.org/features/ToString.
Once something more useful than hashCode is provided by super class - it will be included there.